slicops/ui_api.py

```diff
@@ -19,5 +20,23 @@ class UIAPI(slicops.quest.API):
     Or, raise a `pykern.quest.APIError`.
     """
 
+    def _app_implementation(self, api_args):
+        assert 'app_name' in api_args
+        return importlib.import_module(f'slicops.app.{api_args.app_name}').new_implementation(
```

Comment:
Grab the code from sirepo.util. Default package_path. This should get moved into pykern. Don't hardwire the path.

The module should then define a class that is instantiated with the qcall. This could SlicLet.